test/helpers/with-server.ts

```diff
@@ -1,19 +1,19 @@
 import {promisify} from 'util';
-import * as test from 'ava';
+import {ExecutionContext, Macro} from 'ava'; // eslint-disable-line ava/use-test
```

Comment:
I also noticed that this is quite buggy. Does it also happen when doing `import type`?

Reply:
We just need to fix the eslint-plugin-ava rule.